openpilot/compile.py

```diff
@@ -72,6 +72,11 @@ def compile(dat, output_fn):
     # pass these to thneed
     setattr(prg.clprg, 'op_estimate', prg.op_estimate)
     setattr(prg.clprg, 'prg', prg.prg)
+
+    if getenv("VALIDTEST") == 1:
+      src = re.search(r"=.*\?.*?read_image", prg.prg)
+      if src is not None: raise Exception("Openpilot has valid checks!")
```

Comment:
Aren't there some where it's needed?

Reply:
@geohot 
For openpilot, I think we do not need gated load.
But I have come to this conclusion by brute forcing.
Also, I was assuming that result of the output does not break
"np.testing.assert_allclose(thneed_out, tinygrad_out.numpy())"

Reply:
Like for now no, for the future maybe? If so, maybe to_image_idx is failing or we really need valid check. But I dont really know how to say we "really" need validchecks by just looking at idx idy valid